test/conftest.py

```diff
@@ -9,8 +9,8 @@
 
 logger = logging.getLogger(__name__)
 
-adb_host = "emulator"
-# adb_host = "127.0.0.1"
+# adb_host = "emulator"
```

Comment:
Only thing to consider - https://github.com/spm5065/pure-python-adb/blob/master/docker-compose.yaml 

Perhaps read from environment variables, that way CI + docker can each do their own thing?